build/configs/rtl8730e/scripts/rlx8730e_img2.ld

```diff
@@ -135,6 +135,7 @@ SECTIONS
 
 		*(.vectors)
 		*(.start)
+		*(.ps_boot)

```

Comment:
It would be more suitable to have this commit as a separate PR since it's not directly related to PM

Reply:
It is related to PM, for the arch codes in a32_pm.S, it need to be placed in PSRAM.

Reply:
Okay, I misunderstood it as a general change for relocating multiple/all code sources. Is that not the case?

Reply:
No, only some specific codes will have special marking to be placed to PSRAM section.